mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/kafka/producers/ArenaMigreringTiltaksgjennomforingKafkaProducer.kt

```diff
@@ -20,6 +21,9 @@ class ArenaMigreringTiltaksgjennomforingKafkaProducer(
     )
 
     fun publish(value: ArenaMigreringTiltaksgjennomforingDto) {
+        if (!isTiltakSomSkalPubliseresTilArena(value.tiltakskode)) {
```

Comment:
Tror dette burde styres via `Config` i stedet for `Tiltakstyper.kt`. Da har vi mulighet til å skru på publisering per tiltakstype, per miljø.

Reply:
Sant det